docsrc/index.rst

```diff
@@ -11,6 +11,7 @@
    Basic Tour </basic-tour>
    Advanced Tour </advanced-tour>
    Constrained Bayesian Optimization </constraints>
+   Parameter Types </parameter_types>
```

Comment:
do you want to update the readme too?
I initially just copied the readme into the index.
I actually think it might be nicer to vastly reduce the text in the readme, and just point to this page.

Reply:
I realized that I already removed this section from the README in an earlier version.